tests/Datasets/Attributes/RulesDataset.php

```diff
@@ -0,0 +1,987 @@
+use Spatie\LaravelData\Attributes\Validation\GreaterThanOrEqualTo;
+use Spatie\LaravelData\Attributes\Validation\Image;
+use Spatie\LaravelData\Attributes\Validation\In;
+use Spatie\LaravelData\Attributes\Validation\InArray;
+use Spatie\LaravelData\Attributes\Validation\IntegerType;
+use Spatie\LaravelData\Attributes\Validation\IP;
+use Spatie\LaravelData\Attributes\Validation\IPv4;
+use Spatie\LaravelData\Attributes\Validation\IPv6;
+use Spatie\LaravelData\Attributes\Validation\Json;
+use Spatie\LaravelData\Attributes\Validation\LessThan;
+use Spatie\LaravelData\Attributes\Validation\LessThanOrEqualTo;
+use Spatie\LaravelData\Attributes\Validation\Max;
+use Spatie\LaravelData\Attributes\Validation\Mimes;
+use Spatie\LaravelData\Attributes\Validation\MimeTypes;
+use Spatie\LaravelData\Attributes\Validation\Min;
+use Spatie\LaravelData\Attributes\Validation\MultipleOf;
+use Spatie\LaravelData\Attributes\Validation\NotIn;
+use Spatie\LaravelData\Attributes\Validation\NotRegex;
+use Spatie\LaravelData\Attributes\Validation\Nullable;
+use Spatie\LaravelData\Attributes\Validation\Numeric;
+use Spatie\LaravelData\Attributes\Validation\Password;
+use Spatie\LaravelData\Attributes\Validation\Present;
+use Spatie\LaravelData\Attributes\Validation\Prohibited;
+use Spatie\LaravelData\Attributes\Validation\ProhibitedIf;
+use Spatie\LaravelData\Attributes\Validation\ProhibitedUnless;
+use Spatie\LaravelData\Attributes\Validation\Prohibits;
+use Spatie\LaravelData\Attributes\Validation\Regex;
+use Spatie\LaravelData\Attributes\Validation\Required;
+use Spatie\LaravelData\Attributes\Validation\RequiredIf;
+use Spatie\LaravelData\Attributes\Validation\RequiredUnless;
+use Spatie\LaravelData\Attributes\Validation\RequiredWith;
+use Spatie\LaravelData\Attributes\Validation\RequiredWithAll;
+use Spatie\LaravelData\Attributes\Validation\RequiredWithout;
+use Spatie\LaravelData\Attributes\Validation\RequiredWithoutAll;
+use Spatie\LaravelData\Attributes\Validation\Same;
+use Spatie\LaravelData\Attributes\Validation\Size;
+use Spatie\LaravelData\Attributes\Validation\Sometimes;
+use Spatie\LaravelData\Attributes\Validation\StartsWith;
+use Spatie\LaravelData\Attributes\Validation\StringType;
+use Spatie\LaravelData\Attributes\Validation\Timezone;
+use Spatie\LaravelData\Attributes\Validation\Unique;
+use Spatie\LaravelData\Attributes\Validation\Url;
+use Spatie\LaravelData\Attributes\Validation\Uuid;
+use Spatie\LaravelData\Exceptions\CannotBuildValidationRule;
+use Spatie\LaravelData\Support\Validation\ValidationRule;
+
+function fixature(
+    ValidationRule $attribute,
+    object|string|array $expected,
+    object|string|null $expectCreatedAttribute = null,
+    string $exception = null
+) {
+    return [
+        'attribute' => $attribute,
+        'expected' => $expected,
+        'expectedCreatedAttribute' => $expectCreatedAttribute ?? $attribute,
+        'exception' => $exception,
+    ];
+}
+
+dataset('attributes', function () {
```

Comment:
For this dataset, I had to wait to see if a PR is going to be merged(https://github.com/pestphp/pest/pull/602) in order to have a more readable dataset.